src/shared/sol-iio.c

```diff
@@ -0,0 +1,990 @@
+#include <ctype.h>
```

Comment:
likely these tools are "too smart" and only check files added, and likely you already added it before and is changing the commit, so it wouldn't detect. @anselmolsm ?